openapi/src/main.rs

```diff
@@ -535,6 +543,82 @@ fn gen_generated_schemas(
     }
 }
 
+fn write_out_field(out: &mut String, var_name: &String, var_type: &String, required: bool) {
+    if required {
+        out.push_str(&format!("    pub {}: {},\n", var_name, var_type));
+    } else {
+        out.push_str("    #[serde(skip_serializing_if = \"Option::is_none\")]\n");
+        out.push_str(&format!("    pub {}: Option<{}>,\n", var_name, var_type));
+    }
+}
+
+fn gen_multitype_params(
+    parent_struct_rust_type: &str,
+    param_name: &str,
+    param: &Json,
+    initializers: &mut Vec<(String, String, bool)>,
+    required: bool,
+    state: &mut Generated,
+    out: &mut String,
+) {
+    let member_schema = param["schema"].clone();
+    match gen_member_variable_string(&member_schema) {
+        Ok(type_) => {
+            initializers.push((param_name.into(), type_.clone(), required));
+            write_out_field(out, &param_name.into(), &type_, required);
+        }
+        Err(TypeError::NoType) => {
+            //Weird case, found with anyOf so only case we are handling
+            //at the current moment
+            if let Some(inner_vec) = member_schema["anyOf"].as_array() {
+                for (index, val) in inner_vec.iter().enumerate() {
+                    let new_member_name = format!("{}{}", param_name, index);
```

Comment:
Line in question

Reply:
Yeah, honestly, I had done this at one point.  However, I reverted because I didn't understand the Enum untagged bug at the time, so I thought we had to do something else for the Enums.  Now that I understand that, this might be an easier update.  But not something I can do in the immediate future.  

It's something I'll do the next time I have a free second.  For now, I'll rebase this.